site/app/views/OfficeHoursQueueView.php

```diff
@@ -68,7 +69,8 @@ private function renderPart($viewer, $twig_location) {
           'csrf_token' => $this->core->getCsrfToken(),
           'access_full_grading' => $this->core->getUser()->accessFullGrading(),
           'viewer' => $viewer,
-          'base_url' => $this->core->buildCourseUrl() . '/office_hours_queue'
+          'base_url' => $this->core->buildCourseUrl() . '/office_hours_queue',
+          'student_full' => Utils::getAutoFillData($this->core->getQueries()->getAllUsers())
```

Comment:
Calling database queries in views directly is discouraged.  It would be better to execute the database query in a controller and pass the results into the view.